lib/CrEOF/Geo/String/Parser.php

```diff
@@ -57,29 +52,48 @@ class Parser
     private $nextSymbol;
 
     /**
+     * @var Lexer
+     */
+    private static $lexer;
```

Comment:
Will  creating and using multiple Parser() behave correctly regarding to the static `self::$lexer`